docker/docker.ts

```diff
@@ -234,7 +234,7 @@ async function buildImageAsync(
                 dockerPasswordStdin = true;
             }
         } catch (err) {
-            console.log(`Could not process Docker version (${err})`);
+            pulumi.log.info(`Could not process Docker version (${err})`, logResource);
```

Comment:
This isn't your code so don't block, but should this be an error?

Reply:
yes, it looks like it should.  but i'm not going to touch that now in case something out there is depending on this ;-)